Formula/kdoctor.rb

```diff
@@ -0,0 +1,25 @@
+class Kdoctor < Formula
+  desc "Environment diagnostics for Kotlin Multiplatform Mobile app development"
+  homepage "https://github.com/kotlin/kdoctor"
+  url "https://github.com/Kotlin/kdoctor/archive/refs/tags/v0.0.2.tar.gz"
+  sha256 "e95af8f76fb3a6240758af76b5528e57b41793065354c81bff2fbebe435f22df"
+  license "Apache-2.0"
+  head "https://github.com/Kotlin/kdoctor.git", branch: "master"
+
+  depends_on "openjdk@11"
```

Comment:
Can it work with the most recent jdk?

Reply:
It can,  but java 11 is currently used for development process, so left java 11 just to be sure nothing breaks with new java release

Reply:
ok